VideoWeb/VideoWeb/ClientApp/src/app/services/models/instant-message.ts

```diff
@@ -18,3 +20,14 @@ export class InstantMessage extends ChatResponse {
         }
     }
 }
+
+export class ExtendMessageInfo {
+
+    constructor(from: string, isJudge: boolean) {
```

Comment:
The current from is sufficient enough to determine the intended recipient for now

Reply:
done